_translator-files/po/zh_CN/FAQ.po

```diff
@@ -46,159 +47,161 @@ msgid ""
 "# Jamulus FAQ\n"
 " {:.no_toc}\n"
 msgstr ""
+"# Jamulus 常见问题\n"
+" {:.no_toc}\n"
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:12
 msgid "<details markdown=\"1\">"
-msgstr ""
+msgstr "<details markdown=\"1\">"
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:14
 msgid "<summary>Table of contents</summary>"
-msgstr ""
+msgstr "<summary>目录</summary>"
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:16
 msgid "TOC"
-msgstr ""
+msgstr "TOC"
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:17
 msgid "{:toc}"
-msgstr ""
+msgstr "{:toc}"
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:19
 msgid "</details>"
-msgstr ""
+msgstr "</details>"
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:21
 #, no-wrap
 msgid "**For common problems and their solutions when using Jamulus, see the [Troubleshooting](/wiki/Client-Troubleshooting) page.**\n"
-msgstr ""
+msgstr "**有关使用 Jamulus 时的常见问题及其解决方案，请参阅 [疑难解答](/wiki/Client-Troubleshooting) 页面。**\n"
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:24
 msgid "### Is there a metronome, synchronization, or some other way of keeping in time?"
-msgstr ""
+msgstr "### 是否有节拍器、同步或其他保持时间的方式？"
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:26
 msgid "No. Musicians on a Jamulus Server play in real time together as they would in person. If you want to have a time signal, then there are shared metronome solutions on the Internet you can try. But it’s probably best to just minimise latency so nobody has an overall delay more than about 30-50 ms."
-msgstr ""
+msgstr "不会。Jamulus 服务器上的音乐家可以实时一起演奏，就像他们亲自演奏一样。如果你想有时间信号，那么网上有共享节拍器的解决方案你可以试试。但最好只是尽量减少延迟，这样就没有人有超过 30-50 毫秒的整体延迟。"
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:28
 msgid "Bear in mind also that all participants should follow [The Golden Rule](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together) which will also determine whether you can play in time properly."
-msgstr ""
+msgstr "还请记住，所有参与者都应遵循 [黄金法则](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together) 这也将决定您是否可以使用及时正确。"
```

Comment:
This is an example of one of those links. The part after the hashtag should be translated, using the exact same wording as the heading in `Client-Troubleshooting`. On the other hand, I don't have a clue about how punctuation works in Chinese - there don't seem to be spaces - so I'm not sure if everything should be joined up by hyphens or they're only inserted where there are commas or other punctuation marks. Once I get the website working I'll look into it.

Reply:
Hmm, I'm struggling with this; please don't change these links yet until I figure out how they have to be done in Chinese.